jbpm-workitems/jbpm-workitems-rest/src/main/java/org/jbpm/process/workitem/rest/RESTWorkItemHandler.java

```diff
@@ -905,7 +905,7 @@ protected void addAcceptHeader(HttpRequestBase theMethod,
     protected void addCustomHeaders(String headers,
             BiConsumer<String, String> consumer) {
         for(String h : headers.split(";")) {
-             String[] headerParts = h.split("=");
+             String[] headerParts = h.split(":");
```

Comment:
We can keep previous separator, but forcing the length of the resulting array, just to split with the first separator:
```suggestion
             String[] headerParts = h.split("=", 2);
```

Reply:
@gmunozfe Thank you for pointing it out. I should have thought about backward compatibility.